server/utils/disabledRegions.ts

```diff
@@ -0,0 +1,13 @@
+export default [
```

Comment:
My instinct is it'd be safer to have an enabled list rather than a disabled list. I kinda feel like doing this without an API change to get this info is unsafe in any case, but an enabled list feels less unsafe 🙂